templates/zerver/help/custom-certificates.md

```diff
@@ -15,18 +15,24 @@ If you are absolutely, 100% sure that the Zulip server you are connecting to
 is supposed to have a self-signed certificate, click through the warnings
 and follow the instructions on-screen.
 
-If you are less than 100% sure, contact your server administrator. Accepting
-a malicious self-signed certificate would give a stranger full access to
-your Zulip, including your username and password.
+If you are less than 100% sure, contact your server
+administrator. Accepting a malicious self-signed certificate would
+give a stranger full access to your Zulip, including your username and
+password.  See [the instructions below](#adding-a-certificate) for how
+to permanently trust a self-signed certificate on your system.
```

Comment:
Some browsers like Firefox do not use the system certificate store, so the below instructions might not apply.

Reply:
This is difficult -- hm.  It seems bad form to leave other browsers out, but I don't particularly thing we should be in the business of maintaining instructions for all browsers and platforms.

Saying "These instructions are for Google Chrome, or Zulip Desktop; other Windows browsers will require different steps" seems like a cop-out, but that seems like the most reasonable compromise.

Reply:
Yeah, I’m fine with assuming that the user knows how to use their own web browser—the instructions are mainly for desktop app users. Perhaps we should clarify why we’re telling such users to open Chrome at all?

```
{tab|windows}

On Windows, Zulip Desktop shares the certificate store with
Google Chrome, so you can add certificates to it from inside
Chrome.

1. Open Google Chrome.
…
```

Reply:
Yeah, sounds reasonable.  Reworked to not attempt to offer guidance for anything but the Desktop application.